apps/src/templates/studioHomepages/Sections.jsx

```diff
@@ -58,23 +75,33 @@ const Sections = React.createClass({
             result={this.state.sectionsResult}
             nameOrId={this.state.sectionsResultName}
           />
-          {sections.length > 0 && (
+          {isTeacher && numTeacherSections > 0 && sectionFlow2017 && (
+            <SectionsPage
+              className="sectionPage"
+              teacherHomepage={this.props.teacherHomepage}
+            />
+          )}
```

Comment:
Yes - or at least, most of this should be broken out of `SectionsPage` into something like `SectionsPanel` or `SectionsWidget` so that we can still render `SectionsPage` from sections.js.